docs/src/modules/java/pages/eventsourced.adoc

```diff
@@ -134,48 +134,49 @@ By default, the generated constructor has an `EventSourcedEntityContext` paramet
 
 When more components are added the `AkkaServerlessFactory` is regenerated and you have to adjust the registration from the `Main` class.
 
-=== Unit Tests with autogenerated TestKit
+== Testing the Entity 
 
-We can test this Event Sourced Entity implementation with the ShoppingCartTestkit. This class is generated by AkkaServerless when this project is compiled.
+The following snippet shows how the `ShoppingCartTestKit` is used to test the `ShoppingCart` implementation. AkkaServerless provides two main APIs for unit tests, the `ShoppingCartTestKit` and the `EventSourcedResult`. The former gives us the overall state of the entity. Its state and all the events over the course of all the RPC calls. While the latter only holds the effects produced for each individual RPC call to the Entity.
```

Comment:
let's not say "RPC" here, because it's not Remote when unit testing. We can say just "call".

Reply:
done